spec/features/work_packages/details/inplace_editor/shared_examples.rb

```diff
@@ -95,7 +95,10 @@
     end
 
     it 'focuses the trigger link' do
-      expect(page).to have_selector("#{field.field_selector} #{field.trigger_link_selector}:focus")
+      active_class_name = page.evaluate_script("document.activeElement.className")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.